### ISSUE TITLE
give permission to update the website to the people who occasionally need to do it

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1819]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)| @dreid reported|
|-|-|
|Trac ID|trac#1819|
|Type|enhancement|
|Created|2006-06-14 06:50:50Z|

Specifically this regards people who are likely to do subproject releases, and documentation maintainers.

<details><summary>Searchable metadata</summary>

```
trac-id__1819 1819
type__enhancement enhancement
reporter__dreid dreid
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__invalid invalid
component__website website
keywords__ 
time__1150267850000000 1150267850000000
changetime__1169145784000000 1169145784000000
version__None None
owner__dreid dreid
cc__edsuom cc__dreid cc__oubiwann
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="teknico's avatar" src="https://avatars.githubusercontent.com/u/172521?s=50" width="50" height="50">](https://github.com/teknico)|@teknico set status to `closed`|
|-|-|

Anyone that needs to edit the website, please ask and I'll add you to the wiki_editors group. I just added edsuom (hi, edsuom! ;-) ).

Closing this ticket, since it's a duplicate of [#11245](https://github.com/twisted/twisted/issues/11245).

**Comment by twisted-trac** [comment]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)|@dreid set status to `reopened`|
|-|-|

This isn't a duplicate it refers to modifying the non-wiki portions of the website.  Which is a much smaller group of people.  It is particularly necessary for people who release subprojects and people who may need to update the web documentation.  (I don't think the website documentation should have to wait on a release to be updated for obvious errors such as [#1818](https://github.com/twisted/twisted/issues/1818))

Reopening.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

While I agree that it would be nice to update documentation bugs as soon as they are fixed, I don't think this is as simple as just giving out a few extra commit bits.  To be completely correct, documentation fixes will have to be backported to a release branch and the website updated from that branch.  Otherwise documentation for code changes and feature additions will be mixed in with bug fixes.  It seems like a a simpler idea to just release things more often.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_6"></a>|@glyph set owner to @exarkun<br>@glyph set status to `new`|
|-|-|

I have no idea what this ticket is about.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @dreid|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)<a name="note_8"></a>|@dreid set status to `closed`|
|-|-|

It was about giving subproject maintainers the ability to update the documentation (including API docs) when doing a release of their subproject.
